packages/server/graphql/public/mutations/createPaymentIntent.ts

```diff
@@ -0,0 +1,24 @@
+import {getUserId} from '../../../utils/authorization'
+import {getStripeManager} from '../../../utils/stripe'
+import {MutationResolvers} from '../resolverTypes'
+
+const createPaymentIntent: MutationResolvers['createPaymentIntent'] = async (
+  _source,
+  {},
+  {authToken, dataLoader, socketId: mutatorId}
```

Comment:
+1 `mutatorId` is unused and will probably remain unused as well, can you please clean that up now?